docs/content/drive.md

```diff
@@ -349,7 +349,9 @@ Size of listing chunk 100-1000. 0 to disable. (default 1000)
 
 #### --drive-shared-with-me ####
 
-Only show files that are shared with me
+Instructs rclone to operate on your "Shared with me" folder (where Google Drive lets you access the files and folders have shared with you).
+
+This works both with the "list" (lsd, lsl, etc) and the "copy" commands (copy, sinc, etc), and probably with other commands too.
```

Comment:
`sync` is spelled wrong here!

We should probably be a bit more definitive than "and probably with other commands too." so how about "and with with all other commands too."